<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelDetail</class>
 <widget class="QWidget" name="ChannelDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>212</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="file_name_lineEdit">
       <property name="text">
        <string>file_name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="open_file_toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="sorting_type_label">
       <property name="text">
        <string>Sorting type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortin_type_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>NewLabel1</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="ref_checkBox">
       <property name="text">
        <string>Reference to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ref_comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="filter_checkBox">
       <property name="text">
        <string>Filter raw</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="filter_low_spinBox">
       <property name="maximum">
        <number>6000</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="filter_high_spinBox">
       <property name="maximum">
        <number>30000</number>
       </property>
       <property name="value">
        <number>6000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="extract_wav_pushButton">
     <property name="text">
      <string>Extract Waveform</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="sort_channel_pushButton">
       <property name="text">
        <string>Sort channal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sort_methods_toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
